store/sqlstore/file_info_store.go

```diff
@@ -217,6 +217,36 @@ func (fs SqlFileInfoStore) AttachToPost(fileId, postId string) store.StoreChanne
 	})
 }
 
+func (fs SqlFileInfoStore) AttachMultipleToPost(fileIds []string, postId string, creatorId string) store.StoreChannel {
+	return store.Do(func(result *store.StoreResult) {
+		keys, params := MapStringsToQueryParams(fileIds, "FileId")
+
+		params["PostId"] = postId
+		params["CreatorId"] = creatorId
+
+		sqlResult, err := fs.GetMaster().Exec(
+			`UPDATE
+					FileInfo
+				SET
+					PostId = :PostId
+				WHERE
+					Id IN `+keys+`
+					AND PostId = ''
+					AND CreatorId = :CreatorId`, params)
```

Comment:
Should https://github.com/mattermost/mattermost-server/pull/10094/files#diff-32736de6a4585a5384f7606e57b2792fR211 be corrected with `CreatorId` too?